.changeset/nervous-tables-tie.md

```diff
@@ -0,0 +1,28 @@
+---
+"@comet/cms-admin": minor
+"@comet/cms-api": minor
+---
+
+createImageLinkBlock: Allow overriding name
+
+This allows using two different `ImageLink` blocks in one application.
+
+Perform the following steps to override the name:
+
+1. API: Add the name as second argument in the `createImageLinkBlock` factory:
+
+    ```diff
+    const MyCustomImageLinkBlock = createImageLinkBlock(
+        { link: InternalLinkBlock },
+    +   "MyCustomImageLink"
+    );
+    ```
+
+2. Admin: Set the `name` option in the `createImageLinkBlock` factory:
+
+    ```diff
+    const MyCustomImageLinkBlock = createImageLinkBlock({
+        link: InternalLinkBlock,
+    +   name: "MyCustomImageLink"
+    });
+    ```
```

Comment:
It's kinda strange that these two factories are different (two arguments vs. options object) -> should we unify that in Admin and API?

Reply:
I'm pro unification

Reply:
This would be a breaking change however, so I'd do that for all block factories in a future major release if we decide to change it.